src/verifier/verifier.go

```diff
@@ -2,46 +2,107 @@ package verifier
 
 import (
 	"fmt"
-	tenantMessage "housingSystem/src/messages/tenant"
-	// verifierMessage "housingSystem/src/messages/verifier"
-	"housingSystem/src/shared"
+	"log"
+	"time"
+
+	// verifierMessage "Reactive-Welfare-Housing-System/src/messages/verifier"
+	"Reactive-Welfare-Housing-System/src/messages/distributorMessages"
+	"Reactive-Welfare-Housing-System/src/messages/managerMessages"
+	"Reactive-Welfare-Housing-System/src/messages/sharedMessages"
+	"Reactive-Welfare-Housing-System/src/messages/verifierMessages"
+	"Reactive-Welfare-Housing-System/src/shared"
 
 	"github.com/AsynkronIT/protoactor-go/actor"
-	"github.com/AsynkronIT/protoactor-go/remote"
 )
 
 // Actor The  Verifier Actor
-type Actor struct {
+type verifierActor struct {
 	shared.BaseActor
-	mamager     *actor.PID
-	distributor *actor.PID
-	government  *actor.PID
+	mamager        *actor.PID
+	distributorPID *actor.PID
```

Comment:
There are three *actor.PID, but this one has a name that seems inconsistent with others.